.github/workflows/all-modules-installable.yaml

```diff
@@ -0,0 +1,56 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Validate all modules in index can be added and built
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Install cfbs
+      - name: Install cfbs
+        run: pip3 install cfbs
+
+      # Move the index out of the way
+      - name: Move the index out of the way
+        run: mv cfbs.json index.json
+
+      # Initialize a cfbs project
+      - name: Initialize cfbs project
+        run: cfbs --index index.json init
+
+      # Add masterfiles
+      - name: Add masterfiles
+        run: cfbs --index index.json add masterfiles
+
+      # Add all module
+      - name: Add all modules
+        run: |
+          chmod +x /home/runner/work/build-index/build-index/.github/workflows/add-all-modules.sh
+          /home/runner/work/build-index/build-index/.github/workflows/add-all-modules.sh
```

Comment:
I don't think you need absolute path. I think `.github/workflows/add-all-modules.sh` should do just fine.

Reply:
ack, will try